### PR TITLE
Some Hass.io -> Home Assistant updates

### PR DESCRIPTION
**Pull request by TomBrien**
Update references to Hass.io

### CODE REVIEW

File: docs/troubleshooting/networking.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Having your Home available anywhere and everywhere you go is important, whether
...
 The easiest way is to subscribe to Nabu Casa Cloud for the small monthly fee of $5 US, which will solve all this for you and you can (almost) stop reading here, as well as supporting further development of Home Assistant. Nabu Casa Cloud acts as a "smart" proxy on the internet, tunnelling your frontend in an encrypted manner from your home to your phone, regardless where you are and without requiring opening your home network to inbound traffic from the internet.
 If you don't want to use Nabu Casa Cloud (which is fine, but you should still subscribe and enjoy the warm feeling of supporting Home Assistant), you need HA to be accessible from the internet. This requires opening a port on your router and getting a name for your Home Assistant on the internet. While it is possible to have your HA use Port 8123 internally and have your router do a port-forwarding from say the default https port of 443 to 8123, we recommend you NOT do this for reasons of simplicity which we will explain later. You also need a name for your Home Assistant as hassio.local is a private domain suffix that does not exist on the internet.
```
Comment (schmjop) [inline docs/troubleshooting/networking.md:13]: Replace remaining occurence of hassio.local
```suggestion
If you don't want to use Nabu Casa Cloud (which is fine, but you should still subscribe and enjoy the warm feeling of supporting Home Assistant), you need HA to be accessible from the internet. This requires opening a port on your router and getting a name for your Home Assistant on the internet. While it is possible to have your HA use Port 8123 internally and have your router do a port-forwarding from say the default https port of 443 to 8123, we recommend you NOT do this for reasons of simplicity which we will explain later. You also need a name for your Home Assistant as homeassistant.local is a private domain suffix that does not exist on the internet.

```

**Review by schmjop — CHANGES_REQUESTED** [review]
Replace remaining occurence of hassio.local

File: docs/troubleshooting/networking.md (hunk 1 above)
Comment (TomBrien) [reply docs/troubleshooting/networking.md:13]: Nice spot cheers

**Review by zacwest — APPROVED** [review]

**Review by frenck — APPROVED** [review]
LGTM, thanks 👍

**Pull request merged**